core/arch/arm/kernel/thread_spmc.c

```diff
@@ -45,7 +45,7 @@ struct mem_frag_state {
 #endif
 
 /* Initialized in spmc_init() below */
-static uint16_t my_endpoint_id;
+uint16_t my_endpoint_id;
```

Comment:
declaration missing in some header file

Reply:
The [discussion above](https://github.com/OP-TEE/optee_os/pull/6002#discussion_r1188694570) can affect this part, we might not need to use this variable outside of the file. I would wait with this until we figure it out.

Reply:
I reverted this line.